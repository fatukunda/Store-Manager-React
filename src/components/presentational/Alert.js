import React from 'react';

const Alert = ({ message, className }) => {
    return (
        <div className={className} role="alert">
            {message}
            <button type="button" className="close" data-dismiss="alert" aria-label="Close">
                <span aria-hidden="true">&times;</span>
            </button>
        </div>
    )
}
export default Alert