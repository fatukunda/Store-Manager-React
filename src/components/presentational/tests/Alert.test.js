import React from 'react';
import { shallow } from 'enzyme';
import Alert from '../Alert';
describe('Alert', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<Alert debug />);

        expect(component).toMatchSnapshot();
    });
});