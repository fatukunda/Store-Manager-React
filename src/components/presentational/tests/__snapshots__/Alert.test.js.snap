// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
